builder/lxc/step_lxc_create.go

```diff
@@ -63,6 +63,22 @@ func (s *stepLxcCreate) createFromRootFs(containerName string, config RootFsConf
 	return rootfs, err
 }
 
+func (s *stepLxcCreate) loadSidedisk(containerName, archivePath string, destDir string) (error) {
+	lxcDir := "/var/lib/lxc"
+	destPath := filepath.Join(lxcDir, containerName, "rootfs", destDir)
+
+	commands := make([][]string, 2)
+	commands[0] = []string{"mkdir", "-p", destPath}
+	commands[1] = []string{"tar", "-C", destPath, "-xf", archivePath}
+
+	err := s.SudoCommands(commands...)
+  if err != nil {
+		err = fmt.Errorf("Could not untar sidedisk: %s", err)
```

Comment:
maybe the message could be `Could not load sidedisk:...`, just in the `mkdir -p` fails.